### PR TITLE
RDoc-1053 Studio > Server > Databases > Create New Database > General…

### PR DESCRIPTION
**Pull request by Danielle9897**
… flow

### CODE REVIEW

File: Documentation/4.0/Raven.Documentation.Pages/studio/server/databases/create-new-database/general-flow.markdown [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+
+1. **Replication Factor:**  
+   Set the number of replicas your datatbase should have in the cluster.   
+   The minumun required number is 1. 
+   The maximum number is the cluster size (number of nodes in the cluster).
+
+2. **Dynamic Node Distribution**   
+   Upon a node failure, and if this option is checked,   
+   RavenDB server will automatically replicate the database content to another available node in the cluster  
+   (one that doesn't already contain the the database) so that replication facor is maintained.
```
Comment (arekpalinski) [inline Documentation/4.0/Raven.Documentation.Pages/studio/server/databases/create-new-database/general-flow.markdown]: facor -> factor

File: Documentation/4.0/Raven.Documentation.Pages/studio/server/databases/create-new-database/general-flow.markdown [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+
+3. **Setting replication nodes manually**        
+   Select the specific initial replication nodes from the cluster for the database to replicate to.  
+   If not checked than replication nodes will be selected randomly from the cluster.
```
Comment (arekpalinski) [inline Documentation/4.0/Raven.Documentation.Pages/studio/server/databases/create-new-database/general-flow.markdown]: than -> then

File: Documentation/4.0/Raven.Documentation.Pages/studio/server/databases/create-new-database/general-flow.markdown [hunk 3]
```diff
@@ -0,0 +1,51 @@
...
+{INFO/} 
+<br/>
+### Configure Path
+![Figure 4. Create New Database - Path](images/new-database-general-4.png "Configure Path")
+{INFO: }
+Set the directory path for database data.  
+If not specified then the default value will be as the database name.
```
Comment (arekpalinski) [inline Documentation/4.0/Raven.Documentation.Pages/studio/server/databases/create-new-database/general-flow.markdown]: Please mention what will be physical path (~/Server/Databases/{db-name} ? - there were some changes there recently, please verify that)

File: Documentation/4.0/Raven.Documentation.Pages/studio/server/databases/create-new-database/general-flow.markdown [hunk 4]
```diff
@@ -0,0 +1,51 @@
...
+* Name can't exceed 230 characters  
+* Name can't contain any of: /,\,:,*,?,",<,>,|  
+{INFO/} 
+<br/>
+### Configure Replication
+![Figure 3. Create New Database - Replication](images/new-database-general-3.png "Configure Replication")
+{INFO: }
+
+1. **Replication Factor:**  
+   Set the number of replicas your datatbase should have in the cluster.
```
Comment (arekpalinski) [inline Documentation/4.0/Raven.Documentation.Pages/studio/server/databases/create-new-database/general-flow.markdown]: It's not the number of replicas. See, default value 1 doesn't mean you database will have 1 replica (in result 2 copies of db)

Replication factor determines how many copies of the database will be hold in the cluster

File: Documentation/4.0/Raven.Documentation.Pages/studio/server/databases/create-new-database/general-flow.markdown [hunk 5]
```diff
@@ -0,0 +1,51 @@
...
+{INFO/}
+<br/>
+### Database Name
+![Figure 2. Create New Database - Database name](images/new-database-general-2.png "Enter Database Name")
+{INFO: }
+A database name can be any sequence of characters except for the following rules:  
+
+* Name can't start or end with a '.'  
+* Name can't exceed 230 characters  
+* Name can't contain any of: /,\,:,*,?,",<,>,|
```
Comment (arekpalinski) [inline Documentation/4.0/Raven.Documentation.Pages/studio/server/databases/create-new-database/general-flow.markdown]: also see http://issues.hibernatingrhinos.com/issue/RavenDB-9866

File: Documentation/4.0/Raven.Documentation.Pages/studio/server/databases/create-new-database/general-flow.markdown (hunk 4 above)
Comment (Danielle9897) [reply Documentation/4.0/Raven.Documentation.Pages/studio/server/databases/create-new-database/general-flow.markdown]: As discussed with Pawel - will change to:
Set the number of nodes that will contain this database.

File: Documentation/4.0/Raven.Documentation.Pages/studio/server/databases/create-new-database/general-flow.markdown (hunk 1 above)
Comment (Danielle9897) [reply Documentation/4.0/Raven.Documentation.Pages/studio/server/databases/create-new-database/general-flow.markdown]: done

File: Documentation/4.0/Raven.Documentation.Pages/studio/server/databases/create-new-database/general-flow.markdown (hunk 2 above)
Comment (Danielle9897) [reply Documentation/4.0/Raven.Documentation.Pages/studio/server/databases/create-new-database/general-flow.markdown]: done

File: Documentation/4.0/Raven.Documentation.Pages/studio/server/databases/create-new-database/general-flow.markdown (hunk 5 above)
Comment (Danielle9897) [reply Documentation/4.0/Raven.Documentation.Pages/studio/server/databases/create-new-database/general-flow.markdown]: ok - but not needed to be specified here

Comment (arekpalinski) [reply Documentation/4.0/Raven.Documentation.Pages/studio/server/databases/create-new-database/general-flow.markdown]: right

File: Documentation/4.0/Raven.Documentation.Pages/studio/server/databases/create-new-database/general-flow.markdown (hunk 4 above)
Comment (arekpalinski) [reply Documentation/4.0/Raven.Documentation.Pages/studio/server/databases/create-new-database/general-flow.markdown]: ok

File: Documentation/4.0/Raven.Documentation.Pages/studio/server/databases/create-new-database/general-flow.markdown (hunk 3 above)
Comment (Danielle9897) [reply Documentation/4.0/Raven.Documentation.Pages/studio/server/databases/create-new-database/general-flow.markdown]: done

File: Documentation/4.0/Raven.Documentation.Pages/studio/server/databases/create-new-database/general-flow.markdown [hunk 6]
```diff
@@ -23,25 +23,32 @@ A database name can be any sequence of characters except for the following rules
...
 ### Configure Path
 ![Figure 4. Create New Database - Path](images/new-database-general-4.png "Configure Path")
 {INFO: }
 Set the directory path for database data.  
-If not specified then the default value will be as the database name.
+Use:  
+
+* Full path (e.g. Windows: _C:/MyWork/MyDatabaseFolder_, Linux: _/etc/MyWork/MyDatabaseFolder_ ) - Database will be created in this physical location
+* Relative path (e.g. _MyWork/MyDatabaseFolder_) - Database will be created under RavenDB binaries folder
+* Leave field empty - Database will be created under `Databases` directory in RavenDB binaries folder
```
Comment (arekpalinski) [inline Documentation/4.0/Raven.Documentation.Pages/studio/server/databases/create-new-database/general-flow.markdown]: By default, if the path wasn't absolute one we store data in `RavenData` directory. E.g. `RavenData/Databases/MyDatabase`
https://github.com/ravendb/ravendb/blob/v4.0/src/Raven.Server/Properties/Settings/settings.windows.json

BTW http://issues.hibernatingrhinos.com/issue/RavenDB-9900

File: Documentation/4.0/Raven.Documentation.Pages/studio/server/databases/create-new-database/general-flow.markdown [hunk 7]
```diff
@@ -23,25 +23,32 @@ A database name can be any sequence of characters except for the following rules
...
+
+Note: Path can't start with: $home, '~' or 'appdrive:'
```
Comment (arekpalinski) [inline Documentation/4.0/Raven.Documentation.Pages/studio/server/databases/create-new-database/general-flow.markdown]: Please put this into {NOTE /} block

File: Documentation/4.0/Raven.Documentation.Pages/studio/server/databases/create-new-database/general-flow.markdown (hunk 6 above)
Comment (arekpalinski) [reply Documentation/4.0/Raven.Documentation.Pages/studio/server/databases/create-new-database/general-flow.markdown]: What I think should be clarified here is that the root for relative paths is `DataDir` setting configured in settings.json (Default: `"DataDir": "RavenData"`)

File: Documentation/4.0/Raven.Documentation.Pages/studio/server/databases/create-new-database/general-flow.markdown (hunk 7 above)
Comment (Danielle9897) [reply Documentation/4.0/Raven.Documentation.Pages/studio/server/databases/create-new-database/general-flow.markdown]: done

File: Documentation/4.0/Raven.Documentation.Pages/studio/server/databases/create-new-database/general-flow.markdown (hunk 6 above)
Comment (Danielle9897) [reply Documentation/4.0/Raven.Documentation.Pages/studio/server/databases/create-new-database/general-flow.markdown]: done

**Pull request merged**